cli/ioctl/cmd/account/accountnonce.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 IoTeX
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package account
```

Comment:
I don't think "ioctl account nonce" is necessary. User has no incentive to query a nonce, it is an internal detail.

Reply:
if users want to use the nonce flag when make tx, it is better to have a cmd to query nonce.